ansible/roles/hotpot-installer/tasks/main.yml

```diff
@@ -68,6 +68,13 @@
     dest: "{{ opensds_config_dir }}"
   when: deploy_project != "gelato"
 
+- name: generate certificates
```

Comment:
I think you should create `"{{ opensds_certs_dir }}"` folder before running this command.